content/blog/2022/01/camel-k-release-1-8/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Camel K 1.8 release"
+date: 2022-01-25
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K"]
+preview: "What's new in Camel K 1.8"
+---
+
+<sub><sup>Image copyright: <a rel="nofollow" href='https://stock.pixlr.com/creator/stockunlimited'>stockunlimited</a></sup></sub>
+
+Time for a new release! Time for [Camel K 1.8 version](https://github.com/apache/camel-k/releases/tag/v1.8.0). Keep reading to discover what's new in Camel K! 
+
+This is the set of technologies on which Camel K 1.8 is based:
+
+* Apache Camel K Runtime 1.11.0
+* Apache Camel Quarkus 2.6.0
+* Apache Camel 3.14.0
+* Apache Camel Kamelets 0.7.0
+
+As usual, thanks to Apache Camel, Camel Quarkus and Kamelet Catalog contributors for the great efforts they've put in those new releases as well.
+
+## KEDA support
+
+The most remarkable feature available in this new release, is the support of [KEDA](https://keda.sh/). Thanks to this feature, you will be now able to add **autoscaling features to any kind of event**, not only based on HTTP as we used to have with Knative. Learn more about this fantastic feature reading the dedicated blog, [Camel meets Keda](/blog/2022/01/camel-keda/).
+
+## Faster Operator startup
+
+You already know, Camel K was imprinted with the concept of fast execution. One great work we carried on, was to **reduce the time to startup the operator** in less than a second. Quite remarkable and very welcome in the era of Cloud Native!
+
+## Honor HTTP proxy settings
+
+We had this feature in our TODO list since a while. Finally you will be able to automatically **configure the HTTP proxy settings** in all your `Integrations` by declaring such variables during Camel K Operator installation. Have a look a the [official documentation](/camel-k/next/configuration/http-proxy.html) for more details. 
```

Comment:
Not only in the integrations, but also for all the operator egresses, like downloading Maven artifacts from repositories, pulling / pushing images from container registries, ...

Reply:
Yeah, I thought it was implicit. Makes sense to make it clear.